### ISSUE TITLE
Issue with running Univariate - RESULT: runtime_error:  can't call set_ld_r2_coo before set_mafvec

### ISSUE THREAD

**Issue by jcfoo** [precimed/mixer#10]
Hi,

I was trying to run mixer on Windows but after doing so, I get the error in the subject line, followed by a full crash of MATLAB.  I am using Version 0.9 windows, any idea of what could be the issue?

Thanks in advance!

**Comment by ofrei** [comment]
@jcfoo I'm sorry but currently MiXeR isn't supported on Windows. Any chance of trying it on a linux server?

**Comment by jcfoo** [comment]
Hi, thanks for the reply.  That is too bad- I tried it on Windows first because the MATLAB version 2017a was readily available on a windows machine here - I have access to a linux server but I guess I will need to see if I can get 2017a for Linux. It should be possible but will need some checking :).

**Comment by jcfoo** [comment]
Hi, I am working now on a Linux machine and I get the same issue actually when running the Univariate test.

RESULT: runtime_error:  can't call set_ld_r2_coo before set_mafvec; 

This occurs a few lines after I get the error this:

RESULT: runtime_error:  Variant rs550428710 is not present in .frq files;

Any idea what I should do about it?  I calculated frequency files according to instructions in the readme.

Thanks in advance!

**Comment by ofrei** [comment]
@jcfoo there should be a file called <outname>.bgmglib.log with more details, would it be possible for you to send me this file to me ( oleksandr.frei@gmail.com ) for further investigation?

**Comment by jcfoo** [comment]
Hi all, FYI these issues were resolved. mixer seems to be running to completion withou issue now.
It turns out that the error was caused by different number of variants in bim and frq files.  During the process of creating freq files, one of the files for a specific chromosome somehow was not correctly specified, resulting in a different number of variants.  Once this was corrected everything else appears to be fine.  For everything else I followed the documentation on github. Thanks to @ofrei for the troubleshooting!!
